kurtosis-module/impl/participant_network/mev_boost/mev_boost_launcher.go

```diff
@@ -1,18 +1,61 @@
 package mev_boost
 
-import "github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/services"
+	"github.com/kurtosis-tech/stacktrace"
+)
 
 type MEVBoostLauncher struct {
-	// TODO Analogous to GethLauncher, BesuLauncher, NimbusLauncher, etc.
+	RelayCheck bool
```

Comment:
could we call this `ShouldCheckRelay`? we try to name boolean variables/functions as answers to questions to make them obvious in the code